.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
I think `lint-staged` is too late, we use the standard `lint`, please update.

Here is an example https://github.com/EddieHubCommunity/LinkFree/blob/main/.husky/pre-commit